setup-scripts/install.sh

```diff
@@ -25,11 +25,51 @@ NC='\033[0m' # No Color
 
 BROKER_VERSION="v0.5.2-beta"
 
+# Set the minimum to approx. 6 months prior
+MIN_DOCKER_VERSION=18.09.0
+MIN_DOCKER_COMPOSE_VERSION=1.23.0
+
+# For current versions, don't include patches
+CURRENT_DOCKER_VERSION=18.09
+CURRENT_DOCKER_COMPOSE_VERSION=1.24
+
 # print a message with a color, like msg "my message" $GRAY
 msg () {
   echo -e "${GRAY}${TAG}${NC}:  $2$1${NC}"
 }
 
+# Compares two versions using dot notation (e.g. 1.2.3 < 1.2.4)
+compare_versions () {
```

Comment:
why does this return 0, 1, and 2? (I'm not too familiar with bash)

Reply:
@treygriffith looks like these are return codes. 0 == latest version, 1 == need patch version upgrade but whatever, 2 == you're out of date

Reply:
Yup, it's a bit funky since I believe bash return values are typically used to indicate the status of a command upon exit. An alternative would be assigning the comparison to a variable, then checking the value of the variable for conditional logic.